core/arch/arm/plat-vexpress/conf.mk

```diff
@@ -32,8 +37,6 @@ endif
 
 $(call force,CFG_GENERIC_BOOT,y)
 $(call force,CFG_GIC,y)
-$(call force,CFG_HWSUPP_MEM_PERM_PXN,y)
```

Comment:
Yes. Juno is an A53 and an A57 ? Why does it uses `cortex-a15` for aarch32 implementation ?

Shouldn't we include the **cortex-a53.mk** file instead. Actually, I think I should merge **cortex-a53.mk** and **cortex-a57.mk** into a **cortex-armv8-0.mk** and follow ARMv8 minor versions to distinguish the cores.

Another solution is to simply add here 
```
$(call force,CFG_HWSUPP_MEM_PERM_PXN,y)
$(call force,CFG_HWSUPP_MEM_PERM_WXN,y)
```